server.js

```diff
@@ -98,20 +107,26 @@ function join_custom_room(ws, join_room_json) {
     // Add a prefix to the room id to indicate custom match.
     room_id = "custom_" + room_id
 
+    // Extract room settings from the passed in json.
     var deck_id = join_room_json.deck_id
+    var starting_timer = join_room_json.starting_timer
+    var enforce_timer = join_room_json.enforce_timer
     var player = active_connections.get(ws)
     player.set_deck_id(deck_id)
     var success = false
+
+    // Check whether the player is joining an existing room.
```

Comment:
Can you put this comment inside the if statement and change it to:
// The room the player wants to join already exists.